openapi-clients/forms/base.ts

```diff
@@ -18,7 +18,7 @@ import type { Configuration } from './configuration'
 import type { AxiosPromise, AxiosInstance, RawAxiosRequestConfig } from 'axios'
 import globalAxios from 'axios'
 
-export const BASE_PATH = 'http://localhost:3100'.replace(/\/+$/, '')
+export const BASE_PATH = 'http://localhost:3000'.replace(/\/+$/, '')
```

Comment:
This is only local development dependent. For future reference: if this is the only change, the file probably shouldn't be staged and committed.